inspector.conf.j2

```diff
@@ -4,8 +4,11 @@ debug = true
 transport_url = fake://
 use_stderr = true
 listen_address = ::
+{% if env.INSPECTOR_REVERSE_PROXY_SETUP == "true" %}
```

Comment:
I think you should also limit listen_address to 127.0.0.1 so that the proxied API is not visible.